examples/node/MarketDataSubscription_LongPoll.js

```diff
@@ -1,28 +1,35 @@
 var fs = require('fs');
 var util = require('util');
 var request = require("request");
+var cmdline = require("./cmdline");
 
-var host = process.argv[2] || 'http-api-host';
-var port = 3000;
-var url = 'https://' + host + ':' + port;
-var NUM_REQUEST = 10;   // Number of long-poll requests before unsubscribe
+var NUM_REQUEST = 10; // Number of long-poll requests before unsubscribe.
 var counter = 0;
-var opt = {
-    cert: fs.readFileSync('client.crt'),
-    key: fs.readFileSync('client.key'),
-    ca: fs.readFileSync('bloomberg.crt')
-};
+
+function url()
```

Comment:
Mhm... ok, I'm not a javascript/node expert (not yet at least!) and I don't know all the best practices, but is it expected for actions to be taken at the point when the module is loaded? That would be quite confusing for me personally. At first I didn't notice that is what you want to do. To be honest I like the explicit `parse` call, it is definitely less magical, but I'm ok to adapt if that's the practice in general.